dashboards-observability/public/components/event_analytics/explorer/visualizations/index.tsx

```diff
@@ -86,79 +87,116 @@ export const ExplorerVisualizations = ({
     }
   };
 
+  const [toggleIdToSelectedMap, setToggleIdToSelectedMap] = useState({});
+
+  const onCollapse = (optionId) => {
+    const newToggleIdToSelectedMap = {
+      ...toggleIdToSelectedMap,
+      [optionId]: !toggleIdToSelectedMap[optionId],
+    };
+    setToggleIdToSelectedMap(newToggleIdToSelectedMap);
+  };
+
+  const onChange = (optionId) => {
+    onCollapse(optionId);
+    collapseFn.current(`panel${optionId}`, optionId === '1' ? 'right' : 'left');
+  };
+
   return (
-    <EuiResizableContainer>
-      {(EuiResizablePanel, EuiResizableButton) => (
-        <>
-          <EuiResizablePanel
-            initialSize={17}
-            minSize="300"
-            mode={['collapsible', { position: 'top' }]}
-          >
-            <div className="dscFieldChooser">
-              <Sidebar
-                query={query}
-                explorerFields={explorerFields}
-                explorerData={explorerData}
-                selectedTimestamp={visualizations?.data?.query[SELECTED_TIMESTAMP] || ''}
-                handleOverrideTimestamp={handleOverrideTimestamp}
-                handleAddField={(field: IField) => handleAddField(field)}
-                handleRemoveField={(field: IField) => handleRemoveField(field)}
-                isFieldToggleButtonDisabled={
-                  vis.name === visChartTypes.LogsView
-                    ? isEmpty(explorerData.jsonData) ||
-                      !isEmpty(query[RAW_QUERY].match(PPL_STATS_REGEX))
-                    : true
-                }
+    <EuiResizableContainer onToggleCollapsed={onCollapse} aria-label="aria-label-container">
```

Comment:
Do not use `aria-label` term in `aria-label`. This is what screen-readers will read, which does not make any sense. Read about it before using.